src/Components/Sponsor.js

```diff
@@ -0,0 +1,174 @@
+import "../CSS/Sponsor.css";
+import "../CSS/Officers.css";
+import React, { useState } from "react";
+
+function Sponsor() {
+  const [ContentSelected, setContentSelected] = useState("sponsor");
+  const [contactData, setContactData] = useState({
+    name: "",
+    email: "",
+    company: "",
+    message: "",
+  });
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    const { name, email, company, message } = contactData;
+    window.location.href = `mailto:sweccmailinglist@u.washington.edu?subject=
+    New Contact from ${name}&body=Name: ${name} from ${company} %0D%0AEmail: ${email}%0D%0AMessage: ${message}`;
+    setTimeout(() => {
+      if (document.hasFocus()) {
+        alert(
+          "It seems your email client didn't open automatically. Please send an email to y031125k@gmail.com with your message.",
+        );
+      }
+    }, 500);
+  };
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setContactData((prevState) => ({
+      ...prevState,
+      [name]: value,
+    }));
+  };
+
+  const handleContentChange = (content) => {
+    setContentSelected(content);
+  };
+
+  const Sponsors = () => (
+    <div>
+      <div className="Sponsors-Info">
+        <h2>Our Sponsors</h2>
+        <h3>{new Date().getFullYear()}</h3>
+        <p>
+          We value the supports of our sponsors, and we are thankful for their{" "}
+          <br />
+          supports that promotes many technical SWECC events which enables
+          thousands of <br />
+          students and professionals to communicate and share new insights. We
+          want more <br />
+          to know how those valuable partners have helped us and how they can
+          further support <br />
+          our community as well.
+        </p>
+      </div>
+
+      {/*<div className="Sponsors-Logo">*/}
```

Comment:
Let's delete the commented code.

Reply:
I think placeholder is reasonable to keep in